### DISCUSSION TITLE
单个子域名的多个A记录支持

### DISCUSSION THREAD

**Discussion by ShallowAi** [jeessy2/ddns-go#767] [Q&A]
请问是否支持给单个指定的子域名添加多个不同网卡IP的A记录。
例如 a.example.com 同时解析到 eth0 的 10.0.0.2，eth1的 10.0.0.3，以此类推。

**Comment by jeessy2** [comment]
- 参考 [传递自定义参数](https://github.com/jeessy2/ddns-go/wiki/%E4%BC%A0%E9%80%92%E8%87%AA%E5%AE%9A%E4%B9%89%E5%8F%82%E6%95%B0#aliyun)
 
- 添加同一个域名的多个配置
